browser/css/jsdialogs.css

```diff
@@ -1913,3 +1913,13 @@ kbd,
 .jsdialog-container [id^='info-modal-label'] {
 	display: flex;
 }
+
+/* Custom props */
+#customprops .jsdialog .ui-grid .menubutton {
```

Comment:
Master version is different (and better in my opinion, prevents applying to something different):
https://github.com/CollaboraOnline/online/blob/913f684e3023e1dc43a6c0ef99c8ff1428c74a88/browser/css/jsdialogs.css#L2077
`#customprops` -> `#DocumentPropertiesDialog #customprops`